cmd/eks-node-viewer/main.go

```diff
@@ -73,7 +74,13 @@ func main() {
 	if err != nil {
 		log.Fatalf("creating style, %s", err)
 	}
-	m := model.NewUIModel(strings.Split(flags.ExtraLabels, ","), flags.NodeSort, style)
+
+	out, err := exec.Command("kubectl", "config", "view", "--minify", "-o", "jsonpath='{.clusters[].name}'").Output()
```

Comment:
You can pass a kubeconfig to the tool, in which case this would be incorrect. It should be possible to get the context name from the loaded kubeconfig.